app/src/components/navigation/ProfileMenu.vue

```diff
@@ -61,21 +107,35 @@ const userNavigation: {
             leave-to-class="transform opacity-0 scale-95"
         >
             <MenuItems
-                class="absolute right-0 z-10 mt-2.5 w-32 origin-top-right rounded-md bg-white py-2 shadow-lg ring-1 ring-zinc-900/5 focus:outline-none"
+                class="absolute right-0 z-10 mt-2.5 w-32 origin-top-right rounded-md bg-white py-2 shadow-lg ring-1 ring-zinc-900/5 focus:outline-none dark:bg-zinc-800"
             >
```

Comment:
<img width="174" alt="image" src="https://github.com/user-attachments/assets/e8145d55-ebb4-441f-96ef-99e7c946e36b">

In dark mode, the background colour of the profile menu is the same as the page background. Perhaps it should be one shade lighter?